src/main.rs

```diff
@@ -2,6 +2,19 @@
 async fn main() -> fpm::Result<()> {
     let matches = app(authors(), version()).get_matches();
 
+    // Block of code to run when start-project subcommand is used
+    if let Some(project) = matches.subcommand_matches("start-project") {
+        // project-name => required field (any package Url or standard project name)
+        if let Some(name) = project.value_of("package-name") {
+            // project path => optional
+            let path = project.value_of("package-path");
+            fpm::start_project(name, path).await?;
```

Comment:
Call  `unwrap()` for `package-name` as this is a required field taken care of by the `clap`